book/tutorials/git_and_jupyterlab/jupyter.md

```diff
@@ -185,7 +192,7 @@ just went with a more descriptive label instead of the default value.
 # Change to the website folder, it was created with the `clone`
 cd website
 # Add the remote
-git remote add Hackweek {{ '`{url}.git`'.format(url=website_url) }}
+git remote add Hackweek https://github.com/snowex-hackweek/website2022.git
```

Comment:
we hardcoded this due to trouble doing substitution in backticks i think, maybe we just remove the backticks?

Reply:
As noted in the issue, replacement is not support within code blocks in the MyST substitutions. We could just add a generic placeholder like `__GIT_REMOTE_URL__` or similar